### PR TITLE
Subject

### PR DESCRIPTION
**Pull request by mihirapatel**
Finished making basic sample pages for bills, subjects, and politicians

### CODE REVIEW

**Review by louisaevola — APPROVED** [review]

**Review by katiewu71 — APPROVED** [review]

File: src/components/Politicians/Picture.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import styles from './PoliticianStyle.scss';
+
+export default function Picture() {
+
+  return (
+    <div className={styles.politicianImage}>
+      <img src='https://pbs.twimg.com/profile_images/923274881197895680/AbHcStkl.jpg' class="boom" width= "150px" height="150px"/>
```
Comment (joe-tohdjojo) [inline src/components/Politicians/Picture.js:8]: there is a `class` attribute here. is that meant to be `className`?

**Review by louisaevola — APPROVED** [review]

File: src/components/Subjects/Picture.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import styles from './SubjectStyle.scss';
+
+export default function Picture() {
+
+  return (
+    <div className={styles.subjectImage}>
+      <img src='https://liberal-international.org/wp-content/uploads/2017/06/500px-Pink_Venus_symbol.svg_.png' class="boom" width= "150px" height="150px"/>
```
Comment (joe-tohdjojo) [inline src/components/Subjects/Picture.js:8]: this also has a `class` attribute that seems like it needs to be switched to `className`. are you checking the console or your terminal for errors? there should have been an error pointing this out

**Review by joe-tohdjojo — CHANGES_REQUESTED** [review]
@mihirapatel please fix the changes to the `class` attribute

**Review by mihirapatel — COMMENTED** [review]
fixed class issues

**Comment by joe-tohdjojo** [comment]
Screen Shot 2019-07-24 at 3 58 20 PM

it looks like the logo is not rendering properly

**Comment by oliviadangelo** [comment]
@joseph-tohdjojo I'm seeing the logo rendering properly and am going to merge this just to move things along

**Pull request merged**